qemu/tests/multi_macvtap_devices.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import time
+import random
+from autotest.client import utils
+from autotest.client.shared import error
+from virttest import utils_net, env_process, aexpect
+
+
+def guest_ping(session, dst_ip, count=None, os_type="linux",
+               p_size=1472, timeout=360):
+    """
+    Do ping test in guest
+    """
+    ping_cmd = "ping"
+    if os_type == "linux":
+        if count:
+            ping_cmd += " -c %s" % count
+        ping_cmd += " -s %s %s" % (p_size, dst_ip)
+    else:
+        if not count:
+            ping_cmd += " -t "
+        ping_cmd += " -l %s %s" % (p_size, dst_ip)
+    try:
+        logging.debug("Ping dst vm with cmd: '%s'" % ping_cmd)
+        session.cmd(ping_cmd, timeout=timeout)
+    except aexpect.ShellTimeoutError, err:
+        if count:
+            raise error.TestError("Error during ping guest ip, %s" % err)
```

Comment:
Hi, @xutian
Script works here. If not match if loop, it will just ignore the exception.

We only need add pass in following code:

       except Exception:
            pass